### PR TITLE
Redesign multitasking dialog

### LINKED ISSUE #2684: Redesign menu & dialog – Multitasking (list of tabs)
Old:

New:

Spec:

Just a note that the icon fill for the link icons follows our default value: `#f9f9fa` a80.

Whereas the icon fill for the erase icon remains `#ffffff` a100.

### PR DESCRIPTION
**Pull request by KrzysztofBarszcz**
closes #2684

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt [hunk 1]
```diff
@@ -40,24 +38,10 @@ class SessionViewHolder internal constructor(
...
-
-    private fun updateDrawable(isCurrentSession: Boolean, actionColor: Int, darkColor: Int) {
-        val drawable = AppCompatResources.getDrawable(itemView.context, R.drawable.ic_link) ?: return
-
-        val wrapDrawable = DrawableCompat.wrap(drawable.mutate())
-        DrawableCompat.setTint(wrapDrawable, if (isCurrentSession) actionColor else darkColor)
-
-        textView.setCompoundDrawablesWithIntrinsicBounds(wrapDrawable, null, null, null)
+        val color = Color.rgb(53, 56, 82)
+        if(isCurrentSession) {
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt]: \[ktlint\] WARNING: \[keyword\-spacing\] Missing spacing after "if" in column 11

File: app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt [hunk 2]
```diff
@@ -40,24 +38,10 @@ class SessionViewHolder internal constructor(
...
-    }
-
-    private fun updateDrawable(isCurrentSession: Boolean, actionColor: Int, darkColor: Int) {
-        val drawable = AppCompatResources.getDrawable(itemView.context, R.drawable.ic_link) ?: return
-
-        val wrapDrawable = DrawableCompat.wrap(drawable.mutate())
-        DrawableCompat.setTint(wrapDrawable, if (isCurrentSession) actionColor else darkColor)
-
-        textView.setCompoundDrawablesWithIntrinsicBounds(wrapDrawable, null, null, null)
+        val color = Color.rgb(53, 56, 82)
```
Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt]: \[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt]: \[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.

Comment (sputnikci) [inline app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt]: \[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.

Comment (KrzysztofBarszcz) [reply app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt]: You're right. This color is mentioned in the issue but not defined in `colors.xml`. Shoud I define it there?

**Comment by ekager** [comment]
Hey @KrzysztofBarszcz ! Thanks for your contribution! Looks like there are some merge conflicts. Would you mind rebasing on master and solving these, and then posting screenshots so that our UX person can approve? Thanks so much! :)

**Comment by KrzysztofBarszcz** [comment]
@ekager Hi, I've solved all the conflicts. The problem was, that someone was solving this task in similar time I did and fired a separate PR. After some time, his changes were accepted and the tasks closed by you. Nevertheless, here is my version in case something may be improved

image

**Comment by ekager** [comment]
Ah I'm sorry I didn't realize there were duplicated PRs and I've been trying to get through all the reviews. In the future for UI changes, if you post screenshots and tag our designer, your PRs can be looked at faster. Still appreciate your work and let me know if you want to find more bugs to contribute to! Thanks! :)

**Comment by KrzysztofBarszcz** [comment]
Yes, I would be happy if you could suggest some tasks to work on. BTW, there still is my one PR still open ;)

**Pull request closed without merging**